documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,29 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``,
+for example in a Makefile:
+
+.. code-block:: make
+
+    SIM_ARGS+=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+.. code-block:: bash
+
+    SIM_ARGS=--vcd=anyname.vhd make SIM=ghdl
+
+A VCD file named ``anyname.vcd`` will be generated in the current directory.
+
+:make:var:`SIM_ARGS` can also be used to pass command line arguments related to other waveform formats supported by GHDL like :ref:`Fast Signal Trace (FST) <ghdl:export_waves>`, which is compressed; or :ref:`GHDL Wave File (GHW) <ghdl:export_waves>`, which supports any VHDL type. 
```

Comment:
Since these links all go to the same place, I'm tempted to cut this back to:
```suggestion
:make:var:`SIM_ARGS` can also be used to pass command line arguments related to :ref:`other waveform formats supported by GHDL <ghdl:export_waves>`.

Reply:
LGTM. I think that mentioning FST and GHW would be desirable because those are less known but significantly more efficient formats than VCD. Anyway, I understand that, in the context of cocotb, that might not be so relevant. Hence, I believe both wordings are acceptable.